snapshots/wb/2025-07-01/income_groups.xlsx.dvc

```diff
@@ -2,11 +2,17 @@ meta:
   origin:
     producer: World Bank
     title: Income Classifications
-    citation_full: World Bank - Income classifications (2024).
+    description: |-
+      Every year, the World Bank Group classifies the world’s economies into four income groups: low, lower-middle, upper-middle, and high. These classifications, updated each year on July 1, are based on the previous year’s Gross National Income (GNI) per capita, expressed in U.S. dollars using the [Atlas method](https://datahelpdesk.worldbank.org/knowledgebase/articles/77933-what-is-the-world-bank-atlas-method).
+    citation_full: World Bank - Income classifications (2025).
```

Comment:
I think "Income classifications" may be a good title to keep (it's much more intuitive than the official one), but for the full citation, I suppose the official title of the dataset should be "World Bank Country and Lending Groups", right?

Reply:
Yeah, I guess so. Lending groups are another thing, but they are included in the file and the webpage where they describe the income groups. Thanks!